### PR TITLE
chore: setup wss port

### PR DESCRIPTION
**Pull request by richard-ramos**
This will require updating the fleet to use latest master

### CODE REVIEW

File: ansible/group_vars/go-waku.yml [hunk 1]
```diff
@@ -46,5 +47,6 @@ open_ports_default_chain: 'SERVICES'
 open_ports_list:
   - { port: 80, comment: 'Certbot' }
   - { port: '{{ go_waku_websocket_port }}' }
```
Comment (jakubgs) [inline ansible/group_vars/go-waku.yml]: If we are disabling HTTP websocket we don't have to open the firewall for `go_waku_websocket_port` either right?

Comment (richard-ramos) [reply ansible/group_vars/go-waku.yml]: Right! I commented that line in latest commit

File: ansible/group_vars/go-waku.yml [hunk 2]
```diff
@@ -45,6 +46,7 @@ open_ports_default_comment: '{{ go_waku_service_name }}'
 open_ports_default_chain: 'SERVICES'
 open_ports_list:
   - { port: 80, comment: 'Certbot' }
-  - { port: '{{ go_waku_websocket_port }}' }
+#  - { port: '{{ go_waku_websocket_port }}' }
```
Comment (jakubgs) [inline ansible/group_vars/go-waku.yml]: Why? Just remove it. We don't need it right?

**Review by jakubgs — APPROVED** [review]

**Review by jakubgs — APPROVED** [review]

**Comment by jakubgs** [comment]
Deployed to `go-waku.test`:
```
 > a all -a 'grep secure-port /docker/go-waku/docker-compose.yml' 
node-01.do-ams3.go-waku.test | CHANGED | rc=0 >>
      --websocket-secure-port=443
node-01.gc-us-central1-a.go-waku.test | CHANGED | rc=0 >>
      --websocket-secure-port=443
node-01.ac-cn-hongkong-c.go-waku.test | CHANGED | rc=0 >>
      --websocket-secure-port=443
```

**Pull request merged**